language/stdlib/modules/LibraConfig.move

```diff
@@ -164,6 +167,27 @@ module LibraConfig {
         include SetEnsures<Config>;
     }
 
+    fun disable_reconfiguration(lr_account: &signer) {
```

Comment:
Comments on these functions to document that they are private functions that should only be used by the WriteSet generator?

Reply:
Updated in fixup!